### DISCUSSION TITLE
How to generate animated models compatible with Raylib without Blender ...

### DISCUSSION THREAD

**Discussion by SuperUserNameMan** [raysan5/raylib#4093] [General]
Hello,

I don't know about you, but I'm allergic to Blender because the learning curve is too high.

Raylib is simple and quick to learn. 
My project is simple and quick to implement.
My brain is simple and quickly overflown.

So the external tools i use must be simple and quick to learn too.

Regarding easy modeling and animation, so far i found :

| toolname | fonction | exported format | Note |
|--------------|------------|-----------------------|-----|
| [BlockBench](https://www.blockbench.net/) | low poly 3D editor + UV mapping + texture painting + Animation | animated `.gltf` and `.glb`, and static `.obj` | Animations fully supported by Raylib since https://github.com/raysan5/raylib/pull/4107 and embedded base64 textures in `.gltf` since https://github.com/raysan5/raylib/pull/4112 |
| [Maverick Model 3D](https://clover.moe/mm3d) | 3D modeler + UV mapping + Skeletal Animation | animated `.iqe` and static `.obj`| `.iqe` can be converted to `.iqm` using https://github.com/lsalzman/iqm ; Can also import `.obj` to rig and animate them easily ;  (IQM embedded textures fully supported since https://github.com/raysan5/raylib/pull/4092) |
| [Wings3D](https://www.wings3d.com/) | relatively easy 3D modeler + UV mapping | static `.obj` and static `.gltf` and `.glb` | Easier than Maverick Model 3D for modeling IMHO, but it does not have animation and skeletal function |

**Questions :**

What simple tools do you use for modeling, texturing and animation ?
What is your work flow ?

**Comment by 404alya** [comment]
Blockbench animations are not fully supported as far as I can see. I tried everything to make simple rotation animation work but it seems like raylib only supports basic skeletal animations that blockbench doesn't support.

Edit: nvm I fixed it by checking "Export groups as Armature" option and putting everything in one group.

<img width="913" height="533" alt="image" src="https://github.com/user-attachments/assets/3afe032c-95df-46a7-93fd-982d7da4664d" />

<img width="478" height="279" alt="image" src="https://github.com/user-attachments/assets/f7112fa2-d5b9-4f91-b67e-088ab123e205" />
